make/cluster.sh

```diff
@@ -97,22 +112,31 @@ if printenv K8S_VERSION >/dev/null && [ -n "$K8S_VERSION" ]; then
   k8s_version="$K8S_VERSION"
 fi
 
-# NB: Kind cluster image digests are autogenerated by
-# hack/update-kind-images.sh.
-source "${here}/config/kind/kind_cluster_node_versions.env"
+if [ -n "$update_images" ]; then
+  for img in $images; do
+    sha=$(crane digest "$(cut -d@ -f1 <<<"$img")")
+    if [ "$(cut -d@ -f2 <<<"$img")" != "$sha" ]; then
+      trace sed -i.bak "s|^$img$|$(cut -d@ -f1 <<<"$img")@$sha|" "$0"
+    else
+      printf "${green}${greencheck}Info${end}: $img already uses the latest digest\n" >&2
+    fi
+  done
+  exit 0
+fi
+
```

Comment:
suggestion: I don't understand the motivation for adding this here. We already have [a script](https://github.com/cert-manager/cert-manager/blob/bfcc204c2b385f03490b7ddd5388ef8fbbfa7b51/hack/latest-kind-images.sh) that works just fine for updating kind images, that I think is easier to read than this and which was already simple to use.

I can see you also added a make target for this, but why could we not have a make target to do this using the script we already have today? Why do we need to add more code for this?

```make
# this is untested, it's an example
.PHONY: update-kind-images
update-kind-images: make/config/kind/images.sh

make/config/kind/images.sh: | bin/tools/crane
	CRANE=bin/tools/crane ./hack/latest-kind-images.sh > $@
```

Reply:
We can merge as-is, we don't need to change this. Resolving.